public/src/components/channelManagement/epicTests/epicTestsForm.tsx

```diff
@@ -67,12 +75,23 @@ export interface EpicTest extends Test {
   articlesViewedSettings?: ArticlesViewedSettings;
 }
 
-export type EpicType = 'ARTICLE' | 'LIVEBLOG';
-
 type Props = InnerComponentProps<EpicTest>;
 
 const getEpicTestForm = (epicType: EpicType): React.FC<Props> => {
   const isLiveBlog = epicType === 'LIVEBLOG';
+  const isAppleNews = epicType == 'APPLE_NEWS';
+
+  const createDefaultEpicVariant = (): EpicVariant => ({
+    name: 'CONTROL',
+    heading: undefined,
+    paragraphs: [],
+    highlightedText:
+      'Support the Guardian from as little as %%CURRENCY_SYMBOL%%1 – and it only takes a minute. Thank you.',
```

Comment:
Currently we only have a US channel in AN.
But I think it would be good to support `%%CURRENCY_SYMBOL%%` anyway, for 2 reasons:
1. consistency with other epic tools. Someone may quite reasonably think it needs to be templated.
2. one day we might spin up another regional AN channel

So let's handle the `%%CURRENCY_SYMBOL%%` template in the apple-news backend

Reply:
Would this involve publishing different versions of the article to different channels?

Reply:
Yes, and we can worry about that when we have more channels!